apps/dtlaunch/app-b2.js

```diff
@@ -67,54 +68,60 @@ function draw_icon(p,n,selected) {
       }
     }
     g.drawString(line.trim(),x+36,y+54+lineY*8);
-}
+};
 
-function drawPage(p){
+let drawPage = function(p){
```

Comment:
I tried this in order to have something to delete when fastloading, but there's still ram leaking so don't know if it helped any.

Was inspired by this: https://forum.espruino.com/conversations/379724/?offset=25#comment16739560